install_config/adding_hosts_to_installs.adoc

```diff
@@ -0,0 +1,195 @@
+[[install-config-adding-hosts-to-installs]]
+= Adding Hosts to an Existing Install
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Depending on how your {product-title} cluster was installed, you can add new
+hosts (either nodes or masters) to your installation by using the install tool
+for quick installations, or by using the *_scaleup.yml_* playbook for advanced
+installations.
+
+[[adding-nodes-or-reinstalling-quick]]
+== Adding Hosts Using the Quick Installer Tool
+
+If you used the quick install tool to install your {product-title} cluster, you
+can use the quick install tool to add a new node host to your existing cluster,
+or to reinstall the cluster entirely.
+
+[NOTE]
+====
+Currently, you can not add new master hosts if you used the quick install. 
```

Comment:
Hmm now I can't tell if the info from Andrew and Sam was one way or the other. I certainly intended that you can't add new masters if you've installed using the quick install. @abutcher @smunilla thoughts?